Tests/Opc.Ua.Core.Tests/Security/Certificates/CertificateValidatorTest.cs

```diff
@@ -298,11 +300,114 @@ public async Task VerifySelfSignedAppCertsThrow()
                         var serviceResultException = Assert.Throws<ServiceResultException>(() => certValidator.Validate(new X509Certificate2(cert)));
                         Assert.AreEqual((StatusCode)StatusCodes.BadCertificateUntrusted, (StatusCode)serviceResultException.StatusCode, serviceResultException.Message);
                     }
+
+                    Thread.Sleep(1000);
```

Comment:
In the async tasks it's not good to use Thread.Sleep() instead await Task.Delay() has to be used.

Reply:
indeed... I will fix it with next opportunity.